app/src/main/java/space/taran/arknavigator/mvp/presenter/adapter/ResourcesGridPresenter.kt

```diff
@@ -252,11 +252,36 @@ class ResourcesGridPresenter(
             scoreStorage.getScore(it.id) > 0 || scoreStorage.getScore(it.id) < 0
         }
 
-    fun updateAdapterWithScores() {
+    fun onScoresChangedExternally() {
         sortAllResources()
         sortSelectionAndUpdateAdapter()
     }
 
+    fun onSelectedChangedExternally(selected: List<ResourceId>) =
```

Comment:
What means "externally" here?

Reply:
Selected resources have been changed in another screen